### PR TITLE
chore: Validate pathing to the component pages looks right and is loading from navigation with a single click

### PR DESCRIPTION
**Pull request by Rajdeepc**
## Description
loadIcons path fixed: loadIcons('../../components/icon/spectrum-css-icons.svg');
html in example.njk, components.njk & sidenav.njk are fixed.
resources in GitHub and npm svg rendering issue fixes

## How and where has this been tested?
Chrome

## Screenshots
Screenshot 2023-04-13 at 7 18 53 PM

### CODE REVIEW

File: tools/documentation/_includes/sidenav.njk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% macro renderNavListItem(entry) -%}
-  <li class="spectrum-SideNav-item">
-    <a href="{{ entry.url }}" class="spectrum-SideNav-itemLink {{ 'is-selected' if entry.url == page.url }}">{{ entry.title }}</a>
+  <li class="spectrum-SideNav-item {{ 'is-selected' if entry.url == page.url }}">
```
Comment (castastrophe) [inline tools/documentation/_includes/sidenav.njk:2]: Great catch!

File: tools/documentation/content/components.njk [hunk 2]
```diff
@@ -22,26 +22,26 @@ eleventyComputed:
             <a href="#{{ component.name | slugify }}" class="spectrum-BigSubtleLink">{% if component.title %}{{ component.title }}{% else %}{{ component.name }}{% endif %}</a>
         </h1>
     </header>
-    <table class="spectrum-Table spectrum-Table--sizeM spectrum-Table--quiet">
+    <table class="spectrum-CSSComponent-detailsTable">
```
[the remark this answers is not in the document]
Comment (castastrophe) [reply tools/documentation/content/components.njk:25]: They already exist in the project: components/table. Is that what you're asking?

File: tools/documentation/content/index.njk [hunk 3]
```diff
@@ -3,46 +3,47 @@ layout: pages.njk
...
-        <div class="spectrum-HomeCard">
-            <img class="spectrum-HomeCard-image" src="img/illustration_documentation.svg" alt="Documentation illustation"/>
-            <div class="spectrum-HomeCard-content">
-                <h4 class="spectrum-Heading spectrum-Heading--sizeS">Robust documentation</h4>
-                <p class="spectrum-Body spectrum-Body--sizeM">Spectrum CSS is designed to be used in partnership with Spectrum's detailed usage guidelines.</p>
-                <p class="spectrum-Body spectrum-Body--sizeM">
-                    <a class="spectrum-Link spectrum-Link--quiet" href="https://spectrum.adobe.com/" target="_blank">View Spectrum guidelines</a>
-                </p>
-            </div>
+<div class="spectrum-Site-page spectrum-Typography">
```
[the remark this answers is not in the document]
Comment (Rajdeepc) [reply tools/documentation/content/index.njk:6]: I checked index.html and each component route on this.
spectrum-Site-page is only used in root index.html but for others we don't use it. But definately we can make it unified. Would you like me to make this change?

File: tools/documentation/content/components.njk (hunk 2 above)
[the remark this answers is not in the document]
Comment (Rajdeepc) [reply tools/documentation/content/components.njk:25]: Okay Thanks

File: tools/documentation/content/index.njk (hunk 3 above)
[the remark this answers is not in the document]
Comment (Rajdeepc) [reply tools/documentation/content/index.njk]: Done

File: tools/documentation/_includes/example.njk [hunk 4]
```diff
@@ -2,9 +2,9 @@
   {% if example.name != component.name %}
     <h3 id="{{ example.name | slugify }}" class="spectrum-CSSExample-heading spectrum-Heading spectrum-Heading--sizeS">
       <a href="#{{ example.name | slugify }}" class="spectrum-CSSComponent-link">{{example.name}}</a>
-      {% if example.status -%}
-        <span class="spectrum-StatusLight spectrum-StatusLight--sizeM spectrum-CSSExample-status spectrum-StatusLight--{{ example.status | statusLightVariant }}">
-          {{ example.status }}
+      {% if component.status -%}
+        <span class="spectrum-StatusLight spectrum-StatusLight--sizeM spectrum-CSSExample-status spectrum-StatusLight--{{ component.status | statusLightVariant }}">
+          {{ component.status }}
```
[the remark this answers is not in the document]
Comment (Rajdeepc) [reply tools/documentation/_includes/example.njk]: Done

**Review by castastrophe — APPROVED** [review]

**Pull request merged**